doc-surrealdb_versioned_docs/version-latest/surrealql/functions/database/time.mdx

```diff
@@ -506,6 +510,30 @@ RETURN time::year(d"2021-11-01T08:30:17+00:00");
 
 <br />
 
+## `time::is::leap_year()`
+
+The `time::is::leap_year()` function Checks if given datatime is a leap year.  
+
+```surql title="API DEFINITION"
+time::is::leap_year(datetime) -> bool
+```
+The following example shows this function, and its output, when used in a [`RETURN`](/docs/surrealdb/surrealql/statements/return) statement:
+
+```surql
+-- Checks with current datatime if none is passed
+RETURN time::is::leap_year();
+
+RETURN time::is::leap_year(d"1987-06-22T08:30:45Z");
+[false]
```

Comment:
Shouldn't it just return the plain boolean value. Hence, without the square brackets?